README.md

```diff
@@ -36,3 +36,31 @@ Right now, the state of it is as follows:
   - Inline Styles
   - Template Files
   - Component Style Files
+
+## Getting started
+```bash
+# Create a new nx workspace
+npx create-nx-workspace ng-rspack-test
+# Choose options:
+# - Stack: None
+# - Package-based Monorepo
+# - CI: Do it later
+# - Remote caching: Up to you
+
+# Install the ng-rspack-build package
+npm install @ng-rspack/nx
+
+# Change into project directory and run the app generator
+cd ng-rspack-test
```

Comment:
Move this before the the `npm install @ng-rspack/nx`

Reply:
Done